test/k8sT/manifests/netperf_deployment.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: netperf-server
+  labels:
+    id: netperf-server
+    zgroup: testapp
+spec:
+  containers:
+  - name: netperf
+    image: tgraf/netperf
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: netperf-client
+  labels:
+    id: netperf-client
+    zgroup: testapp
+spec:
+  containers:
+  - name: netperf
+    image: tgraf/netperf
+---
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
The test never actually connects to this service - what is the reason for having it here? Will it be used in forthcoming tests?